Configuration/ProcessModifiers/python/run4_PixelCPEGeneric_cff.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
+
+# This modifier is to run the Pixel Generic CPE algorithm in Run 4 workflows
+
+run4_PixelCPEGeneric =  cms.Modifier()
```

Comment:
What is specific to Run 4 in this modifier? I would imagine one generic `pixelCPEGeneric` to be more useful and not require (much) more effort.

Reply:
@mmusich can comment here

Reply:
I suppose one could remove the `run4` from the modifier name, though hacking the CPE is discouraged for other use-cases and even here is meant to be temporary.